edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/RocksDbOptionsProvider.cs

```diff
@@ -44,19 +44,25 @@ public class RocksDbOptionsProvider : IRocksDbOptionsProvider
         // Once write-ahead logs exceed this size, we will start forcing the flush of
         // column families whose memtables are backed by the oldest live WAL file
         const ulong DefaultMaxTotalWalSize = 512 * 1024 * 1024;
+
+        // Default is to allow RocksDb to keep open as much files as it can, but
+        // to keep RocksDb from over-using system resources, this may be set.
+        const int DefaultMaxOpenFiles = -1;
```

Comment:
Should we change the default to something like 5000, so that customers don't run into it by default?

Reply:
Agreed - [I used 5000 because this was the previous default.](https://github.com/facebook/rocksdb/blob/master/DEFAULT_OPTIONS_HISTORY.md#480-522016)